examples/controls/control.css

```diff
@@ -0,0 +1,12 @@
+#controlDiv {
```

Comment:
This file should be called `controls.css`, and this div `controlsDiv`. Or even better `#controls`. The fact it is a div is an implementation detail.